src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -42,6 +41,8 @@ class Mountainsort5Sorter(BaseSorter):
         "freq_max": 6000,
         "filter": True,
         "whiten": True,  # Important to do whitening
+        "temporary_base_dir": None,
+        "n_jobs_for_preprocessing": 1,
```

Comment:
```suggestion
        "n_jobs_for_preprocessing": -1,
```
Maybe by default it should use all available jobs?